### PR TITLE
dist/debian: add trailer line

### PR DESCRIPTION
**Pull request by tchaikov**
this should address the warnings from dpkg-source and
dpkg-parsechangelog, like:

```
dpkg-source: warning: scylla-package/debian/changelog(l3): found end of file where expected more change data or trailer
```

```
dpkg-parsechangelog: warning:     debian/changelog(l3): found end of file where expected more change data or trailer
```

### CODE REVIEW

File: dist/debian/changelog.template [hunk 1]
```diff
@@ -1,3 +1,5 @@
 %{product}-cqlsh (%{version}-%{release}-%{revision}) %{codename}; urgency=medium
 
   * New release
+
+ -- Israel Fruchter <fruch@scylladb.com> %{timestamp}
```
Comment (fruch) [inline dist/debian/changelog.template:5]: @tchaikov nice of you to sign in my name :)

File: dist/debian/debian_files_gen.py [hunk 2]
```diff
@@ -47,7 +48,13 @@ class DebianFilesTemplate(string.Template):
 shutil.copytree('dist/debian/debian', 'build/debian/debian')
 
 s = DebianFilesTemplate(changelog_template)
-changelog_applied = s.substitute(product=product, version=version, release=release, revision='1', codename='stable')
+now = datetime.datetime.now(tz=datetime.timezone.utc)
+changelog_applied = s.substitute(product=product,
```
Comment (fruch) [inline dist/debian/debian_files_gen.py:52]: this was done on other packages ?

Comment (fruch) [reply dist/debian/debian_files_gen.py:52]: i.e. adding timestamp ?

Comment (fruch) [reply dist/debian/debian_files_gen.py:52]: (we assumed for long time, that the change log is mostly not touched)

**Review by fruch — APPROVED** [review]
LGTM

File: dist/debian/debian_files_gen.py (hunk 2 above)
Comment (tchaikov) [reply dist/debian/debian_files_gen.py:52]: not yet. i created this change in hope to silence the warnings and to comply to the debian packaging policies. will take care of other packages when i have more free cycles.

**Pull request merged**